link/kprobe.go

```diff
@@ -0,0 +1,221 @@
+package link
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/cilium/ebpf"
+)
+
+var (
+	kprobeEventsPath = filepath.Join(tracefsPath, "kprobe_events")
+)
+
+// Kprobe returns a TraceEvent that fires when the given kernel symbol starts
+// executing. See /proc/kallsyms for available symbols. For example, printk():
+//
+//	Kprobe("printk")
+//
+// As of kernel 4.17, the presence of the given symbol is only checked when
+// Attaching a program later. One TraceEvent can be reused to attach an
```

Comment:
Can you document that `TraceEvent.Attach` can be called multiple times instead? (I'm not sure if that is true, but it's kind of implied by the API)

Reply:
Sorry the comment was misleading: document it on the function `TraceEvent.Attach` instead of here and in Kretprobe.